packages/cute/README.md

```diff
@@ -14,5 +13,3 @@ Compatible with Maybeasy and CooperTS.
 
 > yarn add @kofno/cute
 ```
-
-### [API](docs/api/index.md)
```

Comment:
In this case the API link was broken, and I couldn't find the right link. In other cases where the link still works, they might be pointing to old versions of the docs, from before the festive-possum unification. I'm not sure though.